include/kokkos-utils/callbacks/TimerListener.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef KOKKOS_UTILS_CALLBACKS_TIMERLISTENER_HPP
+#define KOKKOS_UTILS_CALLBACKS_TIMERLISTENER_HPP
+
+#include "Kokkos_NumericTraits.hpp"
```

Comment:
Probably a leftover.

Reply:
```suggestion
```

Reply:
Indeed. Good catch! I've deleted this include here.

It seems a related issue is that we're not very consistent yet with "include what you need". So probably best discussed live what we'd want to do.